backend/queries/user_query.py

```diff
@@ -0,0 +1,21 @@
+from models.user import User
+from database import SessionLocal
+
+db = SessionLocal()
+
+class UserQuery():
+    def getUserById(user_id):
+        user = db.query(User).filter(User.id==user_id).first()
+        return user
+    
+    def setEmail(user_id, email):
+        user = db.query(User).filter(User.id == user_id).first()
+        user.email = email
+        db.commit()
+        return user
+
+    def setUsername(user_id, username):
+        user = db.query(User).filter(User.id == user_id).first()
+        user.username = username
+        db.commit()
```

Comment:
Are you sure you are closing a database session here?
Maybe you should use context menager like:
```
from sqlalchemy import create_engine
from sqlalchemy.orm import sessionmaker

# an Engine, which the Session will use for connection
# resources, typically in module scope
engine = create_engine('postgresql://scott:tiger@localhost/')

# a sessionmaker(), also in the same scope as the engine
Session = sessionmaker(engine)

# we can now construct a Session() without needing to pass the
# engine each time
with Session() as session:
    session.add(some_object)
    session.add(some_other_object)
    session.commit()
# closes the session
```
https://docs.sqlalchemy.org/en/14/orm/session_basics.html#using-a-sessionmaker